pkg/scm/git/git.go

```diff
@@ -13,6 +13,8 @@ import (
 	"strconv"
 	"strings"
 
+	log "github.com/golang/glog"
```

Comment:
and if you just mean why didn't i import it as "glog" it's because the wrapper variable is named glog.